src/Compilers/Core/Portable/Symbols/IMethodSymbol.cs

```diff
@@ -263,6 +264,14 @@ public interface IMethodSymbol : ISymbol
         /// </summary>
         IMethodSymbol? PartialImplementationPart { get; }
 
+        /// <summary>
+        /// Returns the implementation attributes for the given method symbol.
+        /// </summary>
+        /// <remarks>
+        /// These attributes are not always emitted as attributes.
```

Comment:
>These attributes are not always emitted as attributes. [](start = 12, length = 54)

These are not attributes at all. If this is what we want to say, we should say that.

Reply:
@AlekseyTs Any recommendation for a good doc comment?

Reply:
> Any recommendation for a good doc comment?

I would remove the remark section completely. And change description to: "Returns the implementation flags for the given method." Any additional information can be found in documentation for the return type, etc. If we want to be very specific, in addition we could say what does this value map to in metadata (table, field). But I don't think this is necessary.

---
In reply to: [569592792](https://github.com/dotnet/roslyn/pull/50560#discussion_r569592792) [](ancestors = 569592792)